site/src/pages/TemplatePage/TemplatePermissionsPage/TemplatePermissionsPage.tsx

```diff
@@ -26,14 +25,11 @@ export const TemplatePermissionsPage: FC<
     context: { templateId: template?.id },
   })
   const { templateACL, userToBeUpdated, groupToBeUpdated } = state.context
-  if (!template || !permissions) {
-    return <Loader />
-  }
```

Comment:
We shouldn't need another loader here, right? We already have the `Suspense` in the parent component with a fallback loader. Also, at this point, we can be certain template is loaded.